src/Checkout.php

```diff
@@ -76,6 +79,27 @@ public function button($label = 'Check out', array $options = [])
         ], $options));
     }
 
+    /**
+     * Redirect to the checkout session.
+     *
+     * @return \Illuminate\Http\RedirectResponse
+     */
+    public function redirect()
+    {
+        return Redirect::to($this->session->url, 303);
```

Comment:
I chose 303 on purpose here because Stripe recommends it in their docs (and @cjavilla-stripe did in one of his video tutorials): https://stripe.com/docs/payments/accept-a-payment?platform=web&ui=checkout#redirect-customers